timelock/Makefile

```
@@ -0,0 +1,20 @@
+default: build
+
+all: build test
+
+test: build
+	cargo test
+	cargo test --features testutils
+
+build:
+	cargo build --target wasm32-unknown-unknown --release -p soroban-timelock-contract
```

Don't need to specify the crate name for single crate workspaces.
```suggestion
	cargo build --target wasm32-unknown-unknown --release
```